vulns/qt/OSV-2020-1671.yaml

```diff
@@ -20,18 +20,9 @@ affects:
     fixed: b3326dc04e20bd1e7a455dd4fe00fba4bf0b7445
```

Comment:
For these, would it be possible to correct the introduced/fixed commits?

Our infrastructure automatically appends `versions` based on these. They'll just come back if the commit references are incorrect.

Reply:
Can I list several versions for the different branches there?

Reply:
Yep, this affects.ranges field is an array. See https://github.com/google/oss-fuzz-vulns/blob/main/vulns/clamav/OSV-2021-40.yaml for an example with 2 ranges in separate branches (we also try to detect cherry-picks automatically if the patch is exactly the same).

Reply:
I added what I know. For some issues it is unclear which exact patch fixed them.

Reply:
Thanks! I'll remove the existing git://code.qt.io/qt/qt5.git entries here since otherwise they will cause the versions you removed to come back.

Let me know if this is incorrect.

Reply:
Still I don't fully understand how this works. Anyway, I'll just keep an eye on it and come back if corrections are needed.

Thank you for your help! :-)

Reply:
In a nutshell, our infrastructure will iterate through all the provided git ranges, and append any git tags that it finds within those ranges to the "versions" list. see e.g. https://github.com/google/oss-fuzz-vulns/commit/8c1affefade76e75b66950db6301963d5e503c5f which was in response to your changes :) 

Thanks for contributing these fixes!

Reply:
Will the [overview page](https://osv.dev/package/OSS-Fuzz/qt) update automatically? It still shows v6.1.0-alpha1 having vulnerabilities [OSV-2020-1577](https://osv.dev/vulnerability/OSV-2020-1577) and [OSV-2020-1671](https://osv.dev/vulnerability/OSV-2020-1671) which both should be fixed there.

Reply:
Yep! That page is auto updated once a day. It looks like it's updated to reflect your changes now.

Reply:
Indeed, thanks!